common/thorhelper/thorsoapcall.cpp

```diff
@@ -2575,6 +2603,8 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
                     else if (keepAlive)
                         persistentHandler->add(socket, &ep, proto);
                 }
+
+                clientSpan->recordSuccess("SoapCall Succeded");
```

Comment:
I don't think this benefits from an associated message.  It doesn't convey any new information.